import_addressbase/import_csv.py

```diff
@@ -173,8 +172,8 @@ def get_action_dicts(filename):
 def import_csv(filename, nodes):
     client = Elasticsearch(nodes)
     # create index if it doesn't exist
-    client.index(index=INDEX_NAME, doc_type=TYPE_TO_INDEX_MAPPING[0][0],
-                 body={})
+    doc_type = list(TYPE_TO_INDEX_MAPPING.keys())[0]
+    client.index(index=INDEX_NAME, doc_type=doc_type, body={})
```

Comment:
We should be careful about this bit - we used to have weird records when we did that unconditionally in the feeder.